marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ds-utils.sjs

```diff
@@ -29,12 +29,17 @@ function throwForbidden(message) {
   fn.error(null, 'RESTAPI-SRVEXERR', Sequence.from([403, message]));
 }
 
+function throwNotFound(message) {
```

Comment:
Pinging @loriwilhelm  for awareness - this is the library where we're defining convenience functions for throwing errors from ML. As more of DHF is converted to use this library, we'll have an easy way of seeing where legit server error are being thrown.